### PR TITLE
Adding FiniteDuration instances

### PR DESCRIPTION
**Pull request by calvinbrown085**
closes https://github.com/typelevel/cats/issues/2517

### CODE REVIEW

**Comment by calvinbrown085** [comment]
@LukaJCB Yes! I'll get those added

**Comment by LukaJCB** [comment]
I'm sorry I think worded myself incorrectly, what I meant was, that we should re-export the instances in `cats.instances` and `cats.implicits` not duplicate them. :)

You can look at other instances in kernel to see how it's done.

**Comment by calvinbrown085** [comment]
@LukaJCB There we go, this should look a little better.

**Comment by calvinbrown085** [comment]
@LukaJCB Thank you, I got those added

**Comment by calvinbrown085** [comment]
Anyone more familiar with this repo than me, why is the CodeCov pipeline failing?

Am I missing some tests?

**Comment by LukaJCB** [comment]
@calvinbrown085 You're not testing any of the instances for their laws. Check the `LawTests` class to see how it's done for other types.

File: kernel/src/main/scala/cats/kernel/instances/all.scala [hunk 1]
```diff
@@ -14,6 +14,7 @@ trait AllInstances
     with EqInstances
     with EitherInstances
     with DurationInstances
+    with FiniteDurationInstances
```
Comment (LukaJCB) [inline kernel/src/main/scala/cats/kernel/instances/all.scala]: This is binary incompatible, you should create another `AllInstancesBinCompat0` trait here, and then mix that into `all`. :)

Comment (calvinbrown085) [reply kernel/src/main/scala/cats/kernel/instances/all.scala]: Ah, that might be what I am missing :) Thank you

Comment (LukaJCB) [reply kernel/src/main/scala/cats/kernel/instances/all.scala]: Of course :)

**Pull request merged**